services/common/boltzmann/client/index.js

```diff
@@ -0,0 +1,207 @@
+'use strict'
```

Comment:
Both the web gateway and registry gateway can use this client to talk to storage.